lib/kayvee.ts

```diff
@@ -3,12 +3,27 @@ _.mixin(require("underscore.deep"));
 
 const deploy_env = process.env._DEPLOY_ENV;
 
+// Encode errors using their properties instead of toJSON()
+function replaceErrors(key, value) {
+  if (value instanceof Error) {
+    var error = {};
+
+    _.each(Object.getOwnPropertyNames(value), (k) => {
+      error[k] = value[k];
```

Comment:
Do we really want all the parameters? Sorry but I'm not sure what it usually looks like